### ISSUE TITLE
[misc] pyenv 2.3.13 is now available

### ISSUE THREAD

**Issue by item4** [CircleCI-Public/cimg-python#182]
Current Python images uses pyenv 2.3.12 and commits on its master branch.
But pyenv 2.3.13 is now available.

ps. Yes I know that current pyenv in image[^1] and 2.3.13 release has only two commits[^2][^3] difference, and these are no related on Python execution.

[^1]: https://github.com/pyenv/pyenv/commit/c8daaa3954a607c135bb93fc5890ba6e3a22e1da
[^2]: https://github.com/pyenv/pyenv/commit/7fe647968d0f023bef10773ef73023caf06c95f7
[^3]: https://github.com/pyenv/pyenv/commit/0ab9683e58b75fd3ee4f5a8b9f3a33d8e3f701bc

**Comment by JalexChen** [comment]
i'm sorry but i'm not sure what the point you're trying to make here is. what is the issue, what needs to be changed, should it be changed, and is there a fix?

**Comment by item4** [comment]
Ok. I will describe current status in text with some (pseudo) terminal logs.

1. You can pull latest-patch versions of Python image such as `cimg/python:3.11.2` or cimg/python:3.10.10`.
2. These images contains pyenv, but these do not match version spec and expected behaviors because pyenv-installer pull master branch. directly.
3. So, when I make this issue, pyenv commit id in docker image was https://github.com/pyenv/pyenv/commit/c8daaa3954a607c135bb93fc5890ba6e3a22e1da , and that commit is not released as pyenv 2.3.13 (so, users must see 2.3.12 when run `pyenv --version`)
4. Python 3.11.2 and 3.10.10 is extended version range since pyenv 2.3.13. but cimg/python was using pyenv 2.3.12 and supporting 3.11.2 and 3.10.10 also.
5. But, it is make debugging very hard! pyenv was not act as 2.3.12, but it was not 2.3.13 also. It increase my dubugging time much longer.

You can get details on `cimg/python@sha256:fa6e64f17819c4678cb58a516620a5c4a659a393c91d5c5a32e21d2f9fae2320` and this steps.

```sh
cd $(pyenv root)

# it show c8daaa3 - Merge pull request #2614 from Afront/3.12.0a5
git log --pretty=format:"%h - %s" -8

# it show 2.3.12
pyenv --version
```

**Comment by felicianotech** [comment]
Pyenv is installed at the latest version whenever a new `cimg/python` image is created: https://github.com/CircleCI-Public/cimg-python/blob/main/Dockerfile.template#L40

We don't make a habit of updating existing images so we wouldn't update the version of Pyenv in an image that already exists.

I hope that explains it. Thank you.
